<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nataly</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow{
	background-color:#1f1f1f;
}

QLabel{
	color:white;
}

QPushButton {
	background-color: #0096FF;
	
	border-radius: 10px;
	color: white;
}


QPushButton:hover {
	background-color: #3063A5;
};




</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>1021</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QStackedWidget{
	background-color:#2e2e2e;
   border-radius:10px;
}


QPushButton {
	background-color: #0096FF;
	border-radius: 10px;
	color: white;

}

QPushButton:!enabled{
	background-color: rgb(119, 118, 123);
color: rgb(154, 153, 150);
}

QPushButton:hover {
	background-color: #3063A5;

}

QCheckBox {
	color:white;
}

QCheckBox::indicator { 
	border: 1px solid; 
	border-radius:7px;
	border-color: white; }

QCheckBox::indicator:checked { 
background-color:#0096FF; 
}

QSlider {
            background-color: #FF3B3B3B;
            color: white;
            border : 0.1em;
            height:  1.5em;
border-radius:7px;
        }

QSlider:hover {
       background-color:rgb(94, 92, 100);
} ;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page1_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SMSA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page2_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>RTST</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Using audio analysis/synthesis from</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="essentia_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/essentia_logo.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="change_theme_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #0096FF;
	border-radius: 15px;
	color: white;

}


QPushButton:hover {
	background-color: #3063A5;

};</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nataly_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/nataly_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1167</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets/nataly_logo.qrc"/>
 </resources>
 <connections/>
</ui>
